tests/components/bmw_connected_drive/conftest.py

```diff
@@ -20,5 +24,11 @@ async def bmw_fixture(monkeypatch):
         AsyncMock(return_value=RemoteServiceStatus({"eventStatus": "EXECUTED"})),
     )
 
+    monkeypatch.setattr(
```

Comment:
Why not patch the returned state from the library and assert the entity state in the state machine after the service call instead?